contributors/devel/development.md

```diff
@@ -329,6 +329,11 @@ git fetch upstream
 git rebase upstream/master
 ```
 
+Please don't use `git merge` instead of the above `fetch` / `rebase`. Merge
```

Comment:
And/or `git pull` (or use `git pull --rebase`). I think most of the time it happens because people use `pull`.